### ISSUE TITLE
Streamline & clearly define our feedback loop

### ISSUE THREAD

**Issue by fharper** [meilisearch/devrel#72]
When to use the GitHub issues, the community Slack, the support email, Twitter, LinkedIn, the forum (GitHub Discussions or in the future, Discourse), private messages...

Once it's decided, we can update the contact page on the website https://github.com/meilisearch/devrel/issues/21#issuecomment-752489322 and move with the Discourse transition #21

**Comment by fharper** [comment]
blocked by #21

**Comment by fharper** [comment]
Duplicate of #127

**Comment by ferdi05** [comment]
process needs to be created. may require assistance from @laurajwilkinson

**Comment by laurajwilkinson** [comment]
@ferdi05 Happy to help. I suggest starting by mapping all the comms channels that DevRel currently have, then noting the use case for each. This will help us identify any areas where we could remove or streamline channels, as well as stating clearly the audience for each, and the type of content (or interaction) to be published there. Let's discuss further once you have this mapped out (and don't hesitate to contact me if you'd like help while carrying this out).

**Comment by ferdi05** [comment]
thanks for your answer @laurajwilkinson . I'd be happy to proceed this way.
I will reach you when the mapping is done.

**Comment by ferdi05** [comment]
@laurajwilkinson you can have a look at #331

**Comment by laurajwilkinson** [comment]
Thanks Ferdinand. The [communication channels mapping](https://github.com/meilisearch/devrel/blob/main/communication/channel_mapping.md) looks to me to be a list of channels and use cases for each, rather than starting with the audiences and how to reach them (maybe I have misunderstood your intention, please say if so). If you're seeking to minimize or make more efficient your use of a large number of channels, this is my suggestion of how to start, so that you end up with the smallest possible number of channels necessary to serve each audience. 

Just wondering also where Stack Overflow and Hacker News fit it - I did a quick search for Meili on [SO](https://stackoverflow.com/search?q=meilisearch) and [HN](https://hn.algolia.com/?q=meilisearch) but no-one seems to be answering consistently from Meili. Does this fall into your remit, or perhaps that of one of the dev teams? Please pass this on if so.

What I've understood from the title of this issue "Streamline & clearly define our feedback loop" and your mention of  "the channel(s) that we will use for support and improving docs, knowledge base" mentioned in [issue 21](https://github.com/meilisearch/devrel/issues/21) is that you're seeking (1) define the channels we use for support, and (2) combine feedback from across our comms channels to improve our [Meili documentation](https://docs.meilisearch.com/). If this understanding is wrong or incomplete, please could you restate and describe your mission?

**Comment by laurajwilkinson** [comment]
Thanks @ferdi05 . I'll start by restating the main themes here for clarity:

1. define the channels we use for support
2. find a way for efficiently extracting user-generated info to push it to various teams at MeiliSearch
3. create some guidelines for using each channel at MeiliSearch
4. combine feedback from across our comms channels to improve our Meili documentation.

Here are my suggestions for your next steps:

0. start a **DevRel handbook** section in Notion (there is already some content in Notion but it's scattered across different sections and it's not clear which parts are current or up-to-date). This will be your internal documentation site for the actions that follow, so that everyone at Meili can access the information.
 1. define the channels we use for support. **Make this one of your pages in the DevRel Handbook**. You've stated that for now, you're not including Stack Overflow, Hacker News, and Reddit as communication channels as we're not yet using them actively enough. But they need to stay on our comms channel list (with a note to say we're not yet actively using them) so that they stay on our horizon and in readiness for the day we want to use them more actively.
2. find a way for efficiently extracting user-generated info to push it to various teams at MeiliSearch. For each channel, start with **the stats each system can provide**, such as Slack Analytics. It won't be very detailed, but it's a useful measure of volume. Next, do a **weekly summary of what topics users were asking about**. If possible, add a subject tag to help with categorising. Put all this information in a new page of your DevRel handbook. Finally, for each Heartbeat update, include the **link to the DevRel handbook page, and highlight 3-5 interesting or persistent topics** that have arisen. (An activity that I will be encouraging us all to do soon is information mapping across Meili, to document who needs to know what and when. Your DevRel user summaries will be a great addition to this).
3. create some guidelines for using each channel at MeiliSearch (maybe have a quick version for everyone at MeiliSearch and a longer version for the DevRel team or anyone at MeiliSearch that actively uses these channels), that'll be great. For the guidelines, it's **essential to write only one version**, otherwise you'll have extra work to do to keep multiple versions synchronised. Make this guide a new page of your DevRel handbook. To achieve the 'quick' version, you could have a list of bullet points where the first sentence of each is the tl;dr, and the rest of the paragraph is the more detailed version.
4. combine feedback from across our comms channels to improve our Meili documentation. The key issue here is being able to **reuse your stats** from point 2 above. Whenever you spot that >1 person is asking about the same subject, (a) **the answer should be in the Meili documentation**, and (b) **the answer should be discoverable** using the words that the user used to describe their issue. Following this logic, you will discover (i) new subjects that are not yet in the docs and need to be flagged to the relevant team for addition, (ii) topics documented but not well understood, where the docs need to be improved, and possibly (iii) topics documented but where the information is out of date or incomplete, in which case the information needs to be updated.

This can seem like a lot to do at the start, but once you have your process up and running, it should get easier. Also, as colleagues learn what you are doing, they can help with different parts of the process. Don't hesitate to schedule a call if you'd like to chat anything through in between our exchanges here.
